gradle-language-server/src/main/java/com/microsoft/gradle/manager/GradleFilesManager.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
```

Comment:
As you haven't implemented services, what's this file for?

Reply:
It's just a standalone file manager to hold the contents of the opening files. Should it be removed from this PR?

Reply:
better to be introduced from the PR related to corresponding features, for easier & clearer reviewing these features later...

Reply:
but I'm ok with it. just make sure to avoid unnessary I/O when you implementing them later.

Reply:
Sure.